### PR TITLE
new sparql by go

### PR DESCRIPTION
**Pull request by GO5IT**
New sparql query is created by Go

### CODE REVIEW

File: sparql/retrieve_entities_v1.sparql [hunk 1]
```diff
@@ -1,3 +1,14 @@
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
```
Comment (sennierer) [inline sparql/retrieve_entities_v1.sparql:1]: same as in the other sparql...Prefixes need to go in the parent

File: start.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
 #memcached -d
-gunicorn main:app -w 4 --reload --timeout=200 --threads=4 -k uvicorn.workers.UvicornWorker --bind 0.0.0.0:5000
\ No newline at end of file
+gunicorn main:app -w 4 --reload --timeout=200 --threads=4 -k uvicorn.workers.UvicornWorker --bind 0.0.0.0:5010
```
Comment (sennierer) [inline start.sh:4]: why change to port 5010?
we cant keep that for the deployment to the cluster. Ingress is listening on port 5000

File: sparql/query_entities_v1.sparql [hunk 3]
```diff
@@ -1,3 +1,11 @@
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
```
Comment (sennierer) [inline sparql/query_entities_v1.sparql:1]: we also need to remove the prefixes here

**Review by sennierer — CHANGES_REQUESTED** [review]
can we also remove the mac specikfic files (.DS_Store) please?
and is the `dump.rdb` file needed?

File: start.sh (hunk 2 above)
Comment (GO5IT) [reply start.sh:4]: Ah, ok. I am using my setting. I need this one for my machine. What would be the best way to be synch with your branch?

File: sparql/retrieve_entities_v2.sparql [hunk 4]
```diff
@@ -1,3 +1,8 @@
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
```
[the remark this answers is not in the document]
Comment (GO5IT) [reply sparql/retrieve_entities_v2.sparql:1]: Ok, but Im a bit confused. all sparql queries in the files in sparql folder contain prefixes, so what is wrong with them?

**Comment by GO5IT** [comment]
I try to remove all problems and clean the code back to the original. I also included new queries which generate better API query results. If there are still issues, please let me know. After this commit, I will fine-tune the queries which Jouni will review. Then, API will be the near-final version of our Deliverable.

**Pull request closed without merging**